### PR TITLE
Add new committee members

### PR DESCRIPTION
**Pull request by chartgerink**
This PR adds the new code of conduct committee members.

This PR is to document this change and ensure all members approve of this update. It's also the space to do a review of the CoC by the new committee members, using our process to document the changes out in the open here on GitHub.

We will add reviewers as they accept the collaboration invitation for this repository.

### CODE REVIEW

**Review by InquisitiveVi — APPROVED** [review]
Addition of new CoC committee members reviewed and approved.

File: pages/coc.tsx [hunk 1]
```diff
@@ -113,6 +113,8 @@ Internal coordinators are always available, external coordinators are available
 |-------------------|-------------------|
 | Chris Hartgerink  | They/them           |
 | Vinodh Ilangovan  | They/them, he/him |
+| Maya Anderson-González  | She/her           |
```
Comment (MayaAndersonGonzalez) [inline pages/coc.tsx:116]: Thank you for adding my name :)

**Review by MayaAndersonGonzalez — COMMENTED** [review]
I hadn't used the review function on github yet,and found the tutorial very useful👍

**Review by andrewypua-projects — APPROVED** [review]

**Comment by chartgerink** [comment]
Thanks for all your reviews!

**Pull request merged**